controllers/categories.controllers.js

```diff
@@ -56,3 +57,12 @@ exports.postReviewComment = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteComment = (req, res, next) => {
+  const comment_id = req.params.comment_id;
+  removeComment(comment_id)
+    .then((result) => {
+      res.status(204).send({ result });
```

Comment:
We shouldn't be sending anything back in the case of a delete, .send() would be suitable here